lib/spack/spack/test/modules/common.py

```diff
@@ -77,6 +92,22 @@ def test_modules_written_with_proper_permissions(mock_module_filename,
         mock_module_filename).st_mode == mock_package_perms
 
 
+@pytest.mark.parametrize('module_type', ['tcl', 'lmod'])
+def test_modules_default_symlink(
+        module_type, mock_packages, mock_module_filename, mock_module_defaults
```

Comment:
I think this is needed to use the mock configuration:
```suggestion
        module_type, mock_packages, mock_module_filename, mock_module_defaults, config
```
and avoid `centos:6` failure on SSL when trying to bootstrap clingo.